library/x509.c

```diff
@@ -1503,10 +1503,16 @@ int mbedtls_x509_info_subject_alt_name(char **buf, size_t *size,
 
             /*
              * dNSName
+             * RFC822 Name
              */
             case MBEDTLS_X509_SAN_DNS_NAME:
+            case MBEDTLS_X509_SAN_RFC822_NAME:
             {
-                ret = mbedtls_snprintf(p, n, "\n%s    dNSName : ", prefix);
+                char * dns_name = "dNSName";
+                char * rfc822_name = "rfc822Name";
+
+                ret = mbedtls_snprintf(p, n, "\n%s    %s : ", prefix,
```

Comment:
Code style checks will want the arguments on the next line aligned after the opening parenthesis.